tests/test.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?><pecan>
+  <outdir>/data/workflows/PEcAn_99000000021</outdir>
```

Comment:
Nit: I think it'd be clearer to name this in a way that's obviously for testing, rather than hardcode a workflow ID that won't have an equivalent in the database. `/tmp/docker_stack_tests/sipnet` or the like?